sentinel_search.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "b58a4e61-67e4-48fd-9bab-3b6319524880",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sentinelsat import SentinelAPI, read_geojson, geojson_to_wkt\n",
+    "from dateparser import parse\n",
+    "import datetime\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9338397b-7c04-4064-bc7a-96b1bc33d1ed",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "7d4b91583ab949c9820ffb545a50770a",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Querying products:   8%|8         | 100/1200 [00:00<?, ?product/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "124f57f9c38e43f7bd8ee0100025562e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Querying products:  72%|#######1  | 100/139 [00:00<?, ?product/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b440a7749c324b0c9f6d994194ab0d2b",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Querying products:  72%|#######1  | 100/139 [00:00<?, ?product/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "5c8c5aad87534c0c8112021d27bf4f1f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Querying products:  89%|########9 | 100/112 [00:00<?, ?product/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "abbafc4b48ad4d4d884c42d4182b3907",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Querying products:  76%|#######6  | 100/131 [00:00<?, ?product/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "d9ba947f9f484e4787c80b2e6459c59b",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Querying products:  83%|########2 | 100/121 [00:00<?, ?product/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# establish connection to Copernicus Scihub\n",
+    "user = \"cta014\"\n",
```

Comment:
I would strongly advice to avoid hardcoding credentials on files that are available on a public GitHub repo. I highly recommend to use environmental variables instead. Check out the following link how to do that if needed: https://towardsdatascience.com/connect-to-databases-using-python-and-hide-secret-keys-with-env-variables-a-brief-tutorial-4f68e33a6dc6